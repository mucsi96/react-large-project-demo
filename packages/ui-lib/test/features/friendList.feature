Feature: Friend List

    Feature Description

    Scenario: Show the list of friends
    Given I am on the page 
    Then I see the list of friends