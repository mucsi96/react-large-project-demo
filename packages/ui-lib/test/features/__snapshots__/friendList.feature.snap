// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show the list of friends 1`] = `
"[
  { name: 'name', value: 'Tom' },
  { name: 'name', value: 'John' },
  { name: 'name', value: 'Alex' }
]"
`;
